ee/tasks/test/subscriptions/test_slack_subscriptions.py

```diff
@@ -430,3 +431,36 @@ def test_async_delivery_retry_on_invalid_blocks_success(
         assert mock_async_client.chat_postMessage.call_count == 3
         assert result.is_complete_success
         mock_sleep.assert_awaited_once_with(1)
+
+
+class TestSlackErrorTruncation(APIBaseTest):
+    def setUp(self) -> None:
+        self.insight = Insight.objects.create(team=self.team, short_id="123456", name="My Test subscription")
+
+    def test_block_for_asset_with_short_error(self) -> None:
+        short_error = "Code: 42. DB::Exception: Received from clickhouse:9000."
+        asset = ExportedAsset.objects.create(
+            team=self.team, insight_id=self.insight.id, export_format="image/png", exception=short_error
+        )
+
+        block = _block_for_asset(asset)
+
+        assert block["type"] == "section"
+        assert short_error in block["text"]["text"]
+        assert "truncated" not in block["text"]["text"]
+        assert len(block["text"]["text"]) < 3000
```

Comment:
```suggestion
        text = block["text"]["text"]
        assert short_error in text
        assert "truncated" not in text
        assert len(text) < 3000
```

Very minor and not worth re-running the full CI for, but this would prevent a bunch of repeated `block[][]` calls and aligns with the other test.